### DISCUSSION TITLE
<title>Image not showing on web page

### DISCUSSION THREAD

**Discussion by leadbewj** [community/community#43631] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

In leadbewj/cv/index.html.   I cannot get my images to show.  When I use it in Pycharm and view locally it works.  I have checked the spelling and the correct case is used.  The image is located in a folder called images/Jeff1.jpg
The code is img src=“images/Jeff1.jpg”.
Everything is in the same case has it should be

**Comment by radar-blip** [comment]
You could try using a forward slash in front of the images folder.

Change:
 `<img scr="images/Jeff1.jpg"`
to
`<img scr="/images/Jeff1.jpg"`

**Reply by leadbewj** [reply]
I’ll try and let you know

**Reply by radar-blip** [reply]
See @jongwooo's answer below for solution.

**Comment by jongwooo** [comment]
@leadbewj Please correct the typo `scr` to `src`.

**Reply by radar-blip** [reply]
Missed that completely.

**Reply by leadbewj** [reply]
the typo was only in this discussion not in the code

**Reply by leadbewj** [reply]
It's fixed.  It was a huge delay on the server updating.
